panel/src/components/Collection/Items.vue

```diff
@@ -156,19 +193,30 @@ export default {
 				fields: this.fields,
 				rows: this.items,
 				selecting: this.selecting,
+				selected: this.selected,
 				sortable: this.sortable
 			};
 		}
 	},
 	methods: {
+		isSelected(item) {
+			return findSelectedIndex(this.selected, item) !== -1;
+		},
 		onDragStart($event, dragText) {
 			this.$panel.drag.start("text", dragText);
 		},
 		onOption(option, item, itemIndex) {
 			this.$emit("option", option, item, itemIndex);
 		},
-		onSelect(event, item, itemIndex) {
-			this.$emit("select", event, item, itemIndex);
+		onSelect(item, itemIndex) {
+			this.$emit("select", item, itemIndex);
+			this.onSelected(item);
+		},
+		onSelected(item) {
+			this.$emit(
+				"selected",
+				updateSelection(this.selected, item, this.selecting)
+			);
```

Comment:
`Discussion:` While `onSelect` moves around the whole item object, I am still wondering if it would be better to have `onSelected` only emit the IDs of selected items.